coreos-meta-translator/trans.py

```diff
@@ -79,7 +79,7 @@ def artifact(i):
         # build the architectures dict
         arch_dict = {"media": {}}
         ensure_dup(input_, arch_dict, "ostree-commit", "commit")
-        generic_arches = ["aliyun", "aws", "azure", "exoscale", "qemu", "metal", "openstack", "vmware", "gcp"]
+        generic_arches = ["aliyun", "aws", "azure", "exoscale", "qemu", "metal", "openstack", "vmware", "gcp", "digitalocean"]
```

Comment:
nit, looks like we were trying to go alphabetical originally, maybe we go back to that?